### PR TITLE
Import German translation stf

### PR DESCRIPTION
**Pull request by dospromptman**

### CODE REVIEW

File: src/objectTranslations/Recurring_Donation__c-de.objectTranslation [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            <translation><!-- Quarterly --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Weekly</masterLabel>
+            <translation><!-- Weekly --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Yearly</masterLabel>
+            <translation><!-- Yearly --></translation>
+        </picklistValues>
```
Comment (davidhabib) [inline src/objectTranslations/Recurring_Donation__c-de.objectTranslation]: remind me why we think these shouldn't be translated.

File: src/objectTranslations/Recurring_Donation__c-de.objectTranslation [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            <translation><!-- Closed --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>None</masterLabel>
+            <translation><!-- None --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Open</masterLabel>
+            <translation><!-- Open --></translation>
+        </picklistValues>
```
Comment (davidhabib) [inline src/objectTranslations/Recurring_Donation__c-de.objectTranslation]: why shouldn't Open Ended Status picklist values get translated?

File: src/objectTranslations/Recurring_Donation__c-de.objectTranslation (hunk 1 above)
Comment (dospromptman) [inline src/objectTranslations/Recurring_Donation__c-de.objectTranslation]: Refer to [this discussion](https://github.com/SalesforceFoundation/Cumulus/pull/2169#discussion_r71413169)

File: src/objectTranslations/Recurring_Donation__c-de.objectTranslation (hunk 2 above)
Comment (dospromptman) [inline src/objectTranslations/Recurring_Donation__c-de.objectTranslation]: Refer to [this discussion](https://github.com/SalesforceFoundation/Cumulus/pull/2169#discussion_r71413169)

**Pull request merged**